app/models/game.rb

```diff
@@ -23,7 +23,6 @@ def reset_piece_players
     pieces.where(is_black: false).update_all(user_id: user_id_white)
   end
 
-  # rubocop:disable Metrics/AbcSize
```

Comment:
Not sure why that wasn't a problem before. Should I try to do something about it other than leave the method disabled?